### ISSUE TITLE
Installer and docs disagree on which port the incoming mail SMTP service lives on

### ISSUE THREAD

**Issue by jdougan** [sandstorm-io/sandstorm#3680]
Docs {1] claim it will be installed on port 30025, the last default installs I did it (v303) put it on port 25.

[1] https://docs.sandstorm.io/en/latest/administering/email/

**Comment by ocdtrekkie** [comment]
I'll examine the installer behavior and then update the docs. My guess is this changed when we updated the installer to attempt to use 443 and such for production installs, but since inbound email is barely supported, nobody updated the docs for it.

**Comment by zenhack** [comment]
Yeah, skimming through the install script it looks like there's some logic that tries to use port 25 if it's available.

**Comment by jdougan** [comment]
Has anything about this been done in some other repo?

**Comment by ocdtrekkie** [comment]
Not yet. It's in my inbox still, but I've had a busy few weeks. The installer and the docs are all both in the main repo.

**Comment by ManasDasri** [comment]
Traced this through `install.sh`. The discrepancy is real and comes from the full-server installer's behavior, not a doc typo:

- `full_server_install()` (the automatic "full server" install path) checks whether port 25 is already bound on the host (`disable_smtp_port_25_if_port_unavailable`). If port 25 is free - which it is on most fresh machines — it sets `SMTP_LISTEN_PORT=25` instead of the 30025 default (`install.sh` lines ~800-804, ~903).
- 30025 only stays in effect if port 25 is already occupied, or if you use the `-d` (dev-install) path, which always uses `DEFAULT_SMTP_PORT=30025` unconditionally.

So the docs (`docs/administering/email.md`, `docs/administering/config-file.md`, `docs/administering/faq.md`) aren't wrong about the *config-file default* - they're just missing the fact that a standard full-server install will silently prefer port 25 over 30025 whenever it can. That's exactly what @jdougan hit on v303.

Suggested fix (doc-only): in `docs/administering/email.md`, change the "By default, Sandstorm's SMTP server runs on port 30025" line to note that full-server installs use port 25 automatically when available, falling back to 30025 otherwise, and add the same clarification to `config-file.md` and the FAQ entry. Happy to open a PR with this wording if that's welcome.

**Comment by ocdtrekkie** [comment]
It *is* welcome, but note that we're now hosting docs on sandstorm.org and the new docs repo is sandstorm-org/docs, so the PR should be submitted there. =)
